README.md

```diff
@@ -21,6 +21,8 @@ developers to quickly build beautiful and consistent interfaces for our users.
 
 ## Development
 
+###### Requires npm v5.7.1 or higher
```

Comment:
Really tiny nit-picky thing but after some thought I think this might be a bit nicer as:

```suggestion
### Prerequisites

- `node@10.x.x`
- `npm@6.x.x`
```

Reply:
I added a new commit similar to this! Let me know if it's okay.

Did you actually want to use node version 10 and npm 6 or were those an example? I assumed they were an example but I can go in and change again. Now worries. 

Thank you for all the tips.

Reply:
Strictly speaking we can get away with what you added but I think I'd like to encourage node 10+ and npm 6+ just to help keep some low hanging edges safe. 😄

Reply:
okay! Updated now! Perhaps this works?